docs/modules/ROOT/pages/api/upgrades.adoc

```diff
@@ -36,6 +36,44 @@ Upgrades the contract code by updating its {class_hash}.
 
 NOTE: This function is usually protected by an xref:access.adoc[Access Control] mechanism.
 
+[.contract]
+[[IUpgradeAndCall]]
```

Comment:
We need to add this to the package README.

Reply:
Good call